lib/libmp/mp.c

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2025 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/init.h>
+
+#if (CONFIG_MP_CAPSFILTER)
```

Comment:
```suggestion
#if CONFIG_MP_CAPSFILTER
```

in order to be consistent amont the library